common/app/views/main.scala.html

```diff
@@ -27,7 +27,7 @@
 }
 
 <!DOCTYPE html>
-<html id="js-context" class="js-off is-not-modern id--signed-out" data-page-path="@request.path">
+<html id="js-context" class="js-off is-not-modern id--signed-out" lang="en" data-page-path="@request.path">
```

Comment:
Can we always assume this? I've seen somewhere that we have some pages in arabic. Is that something that comes down from CAPI?